garage/tf/models/__init__.py

```diff
@@ -1,5 +1,9 @@
 from garage.tf.models.base import Model
 from garage.tf.models.gaussian_mlp_model import GaussianMLPModel
+from garage.tf.models.gaussian_mlp_regressor_model import (
```

Comment:
i think that this should go in the package `garage.tf.regressors` since it's regressor-specific